public/app/model-status-ui.js

```
@@ -1,224 +1,116 @@
-        });
+    let manifest = {};
+    try {
+      const res = await fetch('/app/models-manifest.json', { cache: 'no-cache' });
+      if (res.ok) manifest = await res.json();
+    } catch { /* ignore */ }
+
+    let hasLoading = false;
+    for (const key of this._keys) {
+      const cacheKey = `/app/models/${key}.onnx`;
+      try {
+        const cache = await caches.open('vip-models-v1');
```

ESLint

[error] 'caches' is not defined.

(no-undef)